README.md

```diff
@@ -1 +1,149 @@
-# docker-snowplow
+# Snowplow Docker
+
+## Introduction
+
+This repository contains the Docker images for the following Snowplow components:
+
+- [Scala Stream Collector][ssc]
+- [Stream Enrich][se]
+- [Elasticsearch Loader][es]
+- [S3 Loader][s3]
+
+They are published in the [`snowplow-docker-registry.bintray.io`][registry] docker registry.
+
+## Pulling
+
+You can pull the images from the registry directly:
+
+```bash
+# Scala Stream Collector image
+docker pull snowplow-docker-registry.bintray.io/snowplow/scala-stream-collector:0.10.0
+
+# Stream Enrich image
+docker pull snowplow-docker-registry.bintray.io/snowplow/stream-enrich:0.11.0
+
+# Elasticsearch Loader image
+docker pull snowplow-docker-registry.bintray.io/snowplow/elasticsearch-loader:0.10.0
+
+# S3 Loader image
+docker pull snowplow-docker-registry.bintray.io/snowplow/s3-loader:0.6.0
+```
+
+## Building
+
+Alternatively, you can build them yourself:
+
+```bash
+# All images are based on the base image
+docker build -t snowplow/base:0.1.0 base
+
+# Scala Stream Collector image
+docker build -t snowplow/scala-stream-collector:0.10.0 scala-stream-collector/0.10.0
+
+# Stream Enrich image
+docker build -t snowplow/stream-enrich:0.11.0 stream-enrich/0.11.0
+
+# Elasticsearch Loader image
+docker build -t snowplow/elasticsearch-loader:0.10.0 elasticsearch-loader/0.10.0
+
+# S3 Loader image
+docker build -t snowplow/s3-loader:0.6.0 s3-loader/0.6.0
+```
+
+## Running
+
+Create your own config file filling it according to your setup, you can find examples at the
+following locations:
+
+- [Scala Stream Collector configuration][ssc-config]
+- [Stream Enrich configuration][se-config]
+- [Elasticsearch Loader configuration][es-config]
+- [S3 Loader configuration][s3-config]
+
+Next, you can run a container for each component by mounting your configuration directory:
+
+```bash
+# Scala Stream Collector container
+docker run \
+  -v $PWD/scala-stream-collector-config:/snowplow/config \
+  snowplow/scala-stream-collector:0.10.0 \
+  --config /snowplow/config/config.hocon
+
+# Stream Enrich
+docker run \
+  -v $PWD/stream-enrich-config:/snowplow/config \
+  snowplow/stream-enrich:0.11.0 \
```

Comment:
not sure I understand what you mean?
If you're building locally, you can give any tag, I'm just assuming you have given it the `snowplow/stream-enrich:0.11.0` tag.

Reply:
If you have pulled from our Bintray the path here will be:

> snowplow-docker-registry.bintray.io/snowplow/stream-enrich:0.11.0

Instead of:

> snowplow/stream-enrich:0.11.0

All of the running examples assume you have built the images locally.

Reply:
aah ok, will mention both